docs/glossary.asciidoc

```diff
@@ -423,3 +485,10 @@ A workspace where you build presentations of your live data in <<glossary-canvas
 See
 {kibana-ref}/canvas.html[Create a workpad].
 // end::workpad-def[]
+
+[[glossary-wms]] Web Map Service (WMS) ::
+// tag::wms-def[]
+A layer type in the *Maps* application. WMS stands for Web Map Service.
```

Comment:
Now that the acronym's defined in the title, I don't think this phrase is needed.
```suggestion
A layer type in the *Maps* application. 
```